MicroProfile-Fault-Tolerance/tck-runner/src/test/resources/stable-no-ext-libs-tck-suite.xml

```diff
@@ -11,62 +11,5 @@
             </package>
         </packages>
 
-        <!--below are ignored until fix of PAYARA-3238-->
```

Comment:
I am wondering if PAYARA-3238 being fixed means this can go back in for all payara versions or of this is version dependent.

Reply:
perhaps @Pandrex247 knows?
If not we'll re-enable them. I would like first to ensure the tests all pass before merging this, and again after - test the tests :)

Reply:
Err, Payara-3238 was "fixed" by disabling these tests. Presumably this is version dependent - we'd need to test it.